public/docs/dart/latest/guide/forms.jade

```diff
@@ -518,7 +518,7 @@ figure.image-display
     Recall from the previous section that `ngControl` registered this input box with the
     `NgForm` directive as "name".
 
-    We didn't add the **[`NgForm`](../api/common/index/NgForm-directive.html) directive*
+    We didn't add the **[`NgForm`](../api/angular2.common/NgForm-class.html) directive*
```

Comment:
You didn't change this, but... why does this line have `*`s? Plus, they're mismatched!